src/main/resources/config/application-dev.yml

```
@@ -113,10 +113,12 @@ info:
             - programming_exercise_fail_tour: 'test'
             - programming_exercise_success_tour: 'test'
             - tutor_assessment_tour: 'Patterns in Software Engineering'
-    contact: artemis@xcit.tum.de  #default value, can be overridden if needed
     # Specifies whether text assessment analytics service (TextAssessmentEventResource) is enabled/disabled
     # default value set to true for development environment
     text-assessment-analytics-enabled: true
+    operatorName: Some Artemis Operator # Must be set before starting the application in production. Shown in the about us page and sent to the telemetry service
```

I think we can add an example here, what the operator is would not have been clear to me otherwise 😅 

```suggestion
    operatorName: Some Artemis Operator # Must be set before starting the application in production. Shown in the about us page and sent to the telemetry service (e.g. the name of the university "Technische Universität München")
```

---

Good idea :+1: